frontend/app/src/tx-flows/closeLoanPosition.tsx

```diff
@@ -93,6 +98,33 @@ export const closeLoanPosition: FlowDeclaration<CloseLoanPositionRequest> = {
             </div>,
           ]}
         />
+        {slippageRefund.data && (
+          <TransactionDetailsRow
+            label={
+              <div
+                className={css({
+                  display: "flex",
+                  alignItems: "center",
+                  gap: 4,
+                })}
+              >
+                Slippage refund
+                <InfoTooltip heading="Slippage refund">
+                  Excess collateral was needed to create the desired exposure and accommodate for slippage. This is the
+                  left over amount that has been refunded to your wallet.
+                </InfoTooltip>
+              </div>
+            }
+            value={[
+              <Amount
+                key="start"
+                value={slippageRefund.data}
+                suffix=" BOLD"
+                format="4diff"
```

Comment:
For BOLD, we usually use 2 decimals of precision. Moreover, the slippage refund is a positive quantity by definition, so I would omit the sign. Thus, format should be `2z`.